charts/evm-rollup/files/genesis/geth-genesis.json

```
@@ -38,9 +38,11 @@
         "astriaBridgeAddresses": {{ toPrettyJson .Values.genesis.bridgeAddresses | indent 8 | trim }},
         "astriaFeeCollectors": {{ toPrettyJson .Values.genesis.feeCollectors | indent 8 | trim }},
         "astriaEIP1559Params": {{ toPrettyJson .Values.genesis.eip1559Params | indent 8 | trim }},
-        "astriaSequencerAddressPrefix": "{{ .Values.genesis.sequencerAddressPrefix }}"
+        "astriaSequencerAddressPrefix": "{{ .Values.genesis.sequencerAddressPrefix }}",
         {{- if not .Values.global.dev }}
         {{- else }}
+        "astriaOracleCallerAddress": "{{ .Values.genesis.astriaOracleCallerAddress }}",
```

this might cause issues? The comma might need to be after the `{{- else }}` below because if we are in non dev mode this would leave a trailing comma which is invalid json? Maybe geth doesn't care?
```suggestion
        "astriaSequencerAddressPrefix": "{{ .Values.genesis.sequencerAddressPrefix }}"
        {{- if not .Values.global.dev }}
        {{- else }},
        "astriaOracleCallerAddress": "{{ .Values.genesis.astriaOracleCallerAddress }}",
```